modules/components/src/AdvancedSqonBuilder/filterComponents/TermFilter.js

```diff
@@ -89,7 +89,9 @@ export const TermFilterUI = props => {
         ...currentFieldSqon,
         content: {
           ...currentFieldSqon.content,
-          value: buckets.map(({ key }) => key),
+          value: buckets
+            .filter(({ key }) => key.includes(s.state.searchString))
```

Comment:
there's a new PR recently to apply case insensitive search, maybe split the filter logic from that commit to a common function and reuse here? :) https://github.com/overture-stack/arranger/commit/098e2a95fc3780205dcb610a6ced422e0ba354fd

Reply:
Good point. The same logic is also in the SearchAll filter, and could use that same common function. I'll make that, even if it's out of the scope of this PR.

Reply:
https://github.com/overture-stack/arranger/pull/510